.github/workflows/CI.yml

```diff
@@ -117,3 +117,10 @@ jobs:
           # make git ignore mode changes in files and doesn't include them in the git diff command.
           git config core.fileMode false
           git diff --exit-code
+      - name: Set execute permissions for fsx files
+        run: |
+          chmod +x scripts/eofConvention.fsx
+          chmod +x scripts/shebangConvention.fsx
+          chmod +x scripts/mixedLineEndings.fsx
```

Comment:
@realmarv after you have removed the call to IsExecutable from shebangConventions.fsx, this step above is unnecesary

Reply:
Done

Reply:
Done